sal_interfaces/SALSubsystems.xml

```diff
@@ -1033,20 +1033,20 @@
 
 <Subsystem>
   <Name>FiberSpectrograph</Name>
-  <Description></Description>
-  <Enumeration></Enumeration>
+  <Description>A CSC that manages the Fiber Spectrographs that are used to calibrate the tuneable lasers.</Description>
```

Comment:
@patrickingraham Is the fiber spectrograph used for on AuxTel used for a tunable laser? I thought we only used a tunable laser for the main telescope.